readme.md

```diff
@@ -82,6 +86,10 @@ Use ActiveRecord to create a new `pbj` sandwich and make it belong to the studen
 
 ```rb
 # Your answer...
+geraldo = Student.create(name: "Geraldo")
+pbj = Sandwich.create(type: "pbj", student_id: 1)
+geraldo.save
```

Comment:
also, you don't need to save `geraldo` again, as nothing has changed on his record, just the sandwich.